helm/operator/templates/minio.min.io_tenants.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.13.0
+    controller-gen.kubebuilder.io/version: v0.11.1
```

Comment:
Can you generate with the latest v0.13.0?

Reply:
Fixed, turns out it was a bug on `make regen-crd`